esquery.js

```diff
@@ -274,6 +281,81 @@
             return results;
         }
 
+        /*
+         * Clones a selector AST
+         */
+        function clone(selector) {
```

Comment:
This whole chunk from here down seems to exist purely to translate the old query syntax into the new. Is that necessary? Is the alternative just a breaking change? I think I'd prefer the latter instead of making the PR huge.